authorization-tck/payara-pom.xml

```diff
@@ -0,0 +1,488 @@
+
+                    <!--
+                        Installs the jakarta.authorization-api.jar from GF on the filesystem to the
+                        coordinates expected by the signature test.
+                    -->
+                    <plugin>
+                        <artifactId>maven-install-plugin</artifactId>
+                        <version>3.1.1</version>
+                        <executions>
+                            <execution>
+                                <id>unpack</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>install-file</goal>
+                                </goals>
+                                <configuration>
+                                    <file>${glassfish.root}/payara7/glassfish/modules/jakarta.authorization-api.jar</file>
+                                    <groupId>${sigtest.api.groupId}</groupId>
+                                    <artifactId>${sigtest.api.artifactId}</artifactId>
+                                    <version>${sigtest.api.version}</version>
+                                    <packaging>jar</packaging>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <payara.home>${glassfish.root}/payara7</payara.home>
+
+                                <log.file.location>${glassfish.root}/payara7/glassfish/domains/domain1/logs</log.file.location>
+
+                                <glassfish.systemProperties>
+                                    log.file.location=${glassfish.root}/payara7/glassfish/domains/domain1/logs
+                                    vendor.jakarta.security.jacc.PolicyConfigurationFactory.provider=org.glassfish.exousia.modules.def.DefaultPolicyConfigurationFactory
+                                </glassfish.systemProperties>
+
+                                <glassfish.postBootCommands>
+                                    create-file-user --groups foo:bar --passwordfile ${maven.multiModuleProjectDirectory}/reza.pass reza
+                                    create-file-user --groups Administrator:Manager:Employee --passwordfile ${maven.multiModuleProjectDirectory}/javajoe.pass javajoe
+                                    create-file-user --groups Manager:Employee --passwordfile ${maven.multiModuleProjectDirectory}/javajoe.pass Manager
+                                    create-file-user --groups Employee --passwordfile ${maven.multiModuleProjectDirectory}/j2ee.pass j2ee
+                                </glassfish.postBootCommands>
```

Comment:
I'm not convinced these do anything? I think they're GlassFish Arquillian container things
We probably need to replace these

Reply:
Yeah, these don't do anything.
We need to replace this

Reply:
See https://github.com/luiseufrasio/jakartaee-10-tck-runners/commit/2bea70540067e6c47fde243233e6c21d61d0bffc